include/wx/generic/dirctrlg.h

```diff
@@ -20,13 +20,16 @@
 #include "wx/dialog.h"
 #include "wx/dirdlg.h"
 #include "wx/choice.h"
+#include "wx/menu.h"
```

Comment:
There is no need to include this header here, forward-declaring `wxMenu` is enough in this header.